GetandTake/Pages/Categories/Images.cshtml.cs

```diff
@@ -2,22 +2,20 @@
 using GetandTake.Services.Abstract;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace GetandTake.Pages.Categories
+namespace GetandTake.Pages.Categories;
+
+public class ImagesModel : PageModel
 {
-    public class ImagesModel : PageModel
-    {
-        private readonly ICategoryService _categoryService;
+    private readonly ICategoryService _categoryService;
 
-        public IEnumerable<Category> Categories { get; private set; }
+    public IEnumerable<Category> Categories { get; private set; }
 
-        public ImagesModel(
-            ICategoryService categoryService)
-        {
-            _categoryService = categoryService;
-        }
-        public async Task OnGet()
-        {
-            Categories = await _categoryService.GetAllAsync();
-        }
+    public ImagesModel(ICategoryService categoryService)
+    {
+        _categoryService = categoryService;
+    }
+    public async Task OnGet()
```

Comment:
Let's add empty line before this method to avoid it overlapping with constructor